myuw/dao/enrollment.py

```diff
@@ -18,18 +21,62 @@
     "SENIOR": 4,
     "GRADUATE": 5,
 }
+DEFAULT_CLASS_CODE = 6
+logger = logging.getLogger(__name__)
 
 
-DEFAULT_CLASS_CODE = 6
+def get_all_enrollments():
```

Comment:
line 36-27 are not necessary as enrollment_search_by_regid has been tested in uw_sws